.github/workflows/NuGetCD.yml

```diff
@@ -64,7 +64,7 @@ jobs:
     # create version
     - name: Bump version and push tag
       id: tag_version
-      uses: loekd/github-tag-action@v6.1
+      uses: loekd/github-tag-action@v6.2
```

Comment:
@loekd I see it uses the [forked](https://github.com/loekd/github-tag-action) version. I'm not too familiar with GitHub actions (more with ADO) so I wonder what's needed to pick up the latest version which is [6.2](https://github.com/mathieudutour/github-tag-action/releases/tag/v6.2).